examples/with-stomp/package.json

```diff
@@ -7,9 +7,9 @@
     "start": "next start"
   },
   "dependencies": {
+    "@stomp/stompjs": "^5.4.2",
     "next": "latest",
     "react": "^16.7.0",
-    "react-dom": "^16.7.0",
-    "@stomp/stompjs": "^5.4.2"
+    "react-dom": "^16.7.0"
```

Comment:
I don't think this order changes are expected 🤔

Reply:
It's the order they would have been placed in when installed using `yarn add foo` or `npm i foo`